CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+# Contributing to KubeVirt-Ansible
+
+## Intro
+
+In addition for allowing deployment of KubeVirt via Ansible,
+the KubeVirt-Ansible project is open for any contributions from
+Additional projects which aims to integrate with KubeVirt, such 
+as a specific storage or network project.
+
+The integration will be in the form of Ansible roles per component
+which might later be integrated into a main Ansible Playbook Bundle.
+
+Information on how to contribute will follow in the next sections.
+
+## Contributing new Ansible roles  
+
+Contributing to KubeVirt-Ansible should be as simple as possible. 
```

Comment:
In the section do you think we can have a series of steps on how to contribute a role? For example, 
```bash
kubevirt/playbooks/
├── kubevirt.yaml
├── group_vars
│   └── all.yaml
├── roles
│   └── <new_component>.yaml
│   │   ├── defaults
│   │   │   └── main.yaml
│   │   ├── files
│   │   └── tasks
│   │       └── main.yaml

```
1) The new component will be a subdirectory in roles.
2) Add your cluster resources to ```tasks/main.yaml```
3) Add a flag to enable/disable your component to  ```group_vars/all.yaml```.  ```my_component: false```

This may change with the reorg of the repo, but maybe we can do an initial sketch based on what's currently in the repo.

Reply:
+1, the question is when do we expect the re-org to happen, it can be quite different from the current status right?

Reply:
the discussion is happening in parallel. Hopefully we can get consensus on a plan by the end of the week.

Reply:
I think that we should use `tags` for enabling/disabling roles: http://docs.ansible.com/ansible/latest/playbooks_tags.html